api/db_handlers.js

```diff
@@ -1,47 +1,66 @@
-function addIssue (client, obj, callback) {
-    var timeMS = Date.parse(obj.created_at);
+
+
+function addUser (client, obj, callback) {
     var multi  = client.multi();
 
-    multi.hmset("issue:" + obj.id, obj);
-    multi.zadd("issues", timeMS, obj.id);
+    multi.hmset("user:" + obj.username, obj);
+    multi.sadd("users", obj.username);
     multi.exec(function (errors, replies) {
         return callback(errors, replies);
     });
 }
 
-function deleteIssueById (client, issueId, callback) {
-    var multi = client.multi();
+function getUserByUsername (client, username, callback) {
+    client.hgetall("user:" + username, callback);
+}
 
-    multi.zrem("issues", issueId);
-    multi.del("issue:" + issueId);
-    multi.exec(function (errors, replies) {
-        return callback(errors, replies);
-    });
+
+function addIssue (client, obj, callback) {
+    client.hmset("issue:" + obj.id, obj, callback);
 }
 
-function addUser (client, obj, callback) {
+function addIssueToUserList (client, username, issueId, lastUpdateTime, callback) {
+    var timeMS = Date.parse(lastUpdateTime);
+
+    client.zadd(username + ":issues", timeMS, issueId, callback)
+}
+
+function addIssuesByUsername (client, username, issuesArray, callback) {
     var multi  = client.multi();
 
-    multi.hmset("user:" + obj.id, obj);
-    multi.sadd("users", obj.id);
+    issuesArray.forEach(function (issue) {
+        addIssue(multi, issue);
+        addIssueToUserList(multi, username, issue.id, issue.updated_at, function(err, reply){console.log(err, reply)});
+    })
     multi.exec(function (errors, replies) {
         return callback(errors, replies);
     });
 }
 
-function deleteUserById (client, userId, callback) {
-    var multi = client.multi();
+function getIssueListByUsername (client, username, callback) {
+    client.zrange(username + ":issues",0,-1, callback); //err, data
+}
 
-    multi.srem("users", userId);
-    multi.del("user:" + userId);
-    multi.exec(function (errors, replies) {
-        return callback(errors, replies);
+function getIssuesByList (client, issueList, callback) {
+    var multi = client.multi();
+    issueList.forEach(function (issueId) {
+      multi.hgetall("issue:" + issueId)
+    });
+    multi.exec(function (err, replies) {
+      return callback(err, replies);
     });
 }
 
+function getIssuesByUsername (client, username, callback) {
+    getIssueListByUsername(client, username, function(err, data) {
+        getIssuesByList (client, data, callback);
```

Comment:
Can you please delete the space between the function and the brackets to keep it inline with how the rest of the functions are invoked?